docs/customization/user_provider.md

```diff
@@ -13,7 +15,41 @@ php spark shield:model UserModel
 
 The class name is optional. If none is provided, the generated class name would be `UserModel`.
 
-After creating the class, set the `$userProvider` property in **app/Config/Auth.php** as follows:
+## Customizing Your UserModel
+
+Customize your model as you like.
+
+If you add attributes, don't forget to add the attributes to the `$allowedFields`
+property.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use CodeIgniter\Shield\Models\UserModel as ShieldUserModel;
+
+class UserModel extends ShieldUserModel
+{
+    protected function initialize(): void
+    {
+        parent::initialize();
+
+        $this->allowedFields = [
+            ...$this->allowedFields,
+            'first_name', // Added
+            'last_name',  // Added
+        ];
+    }
+}
+```
+
+## Configuring to Use Your UserModel
```

Comment:
How about putting "Configuring to Use Your UserModel" after "Creating Your Own UserModel"?

Reply:
Good! Done.

Reply:
Thanks.